cookbooks/cdo-mysql/templates/default/proxysql.cnf.erb

```diff
@@ -82,6 +82,13 @@ mysql_variables=
   # https://github.com/sysown/proxysql/wiki/Global-variables#mysql-ping_timeout_server
   # [Milliseconds, default 200]
   ping_timeout_server=1000
+
+  # Prevent Connection Attempt spikes - a negative feedback cycle where a short period of high DB Load due to high usage
+  # or inefficient queries increases query latency, which increases the likelihood that ProxySQL needs to open new
+  # connections to execute new queries, which in turn further degrades query latency due to CPU load to service
+  # new connection attempts.
+  # https://proxysql.com/documentation/global-variables/mysql-variables/#mysql-throttle_connections_per_sec_to_hostgroup
+  throttle_connections_per_sec_to_hostgroup = <%= throttle_connections_per_sec_to_hostgroup  %>
```

Comment:
Does this not need to be
```suggestion
  throttle_connections_per_sec_to_hostgroup = <%= @throttle_connections_per_sec_to_hostgroup  %>
```

?

Reply:
Or, even better:

```suggestion
  throttle_connections_per_sec_to_hostgroup = <%= @throttle_connections_per_sec_to_hostgroup %>
```

Reply:
oops. good catch! Fortunately the existing syntax worked, but how would you feel about just referencing the attribute directly?

https://github.com/code-dot-org/code-dot-org/blob/a145c11a95cf7b1470f10c6082f5efd6eb13b386/cookbooks/cdo-mysql/templates/default/proxysql.cnf.erb#L91

Reply:
I successfully provisioned a "full stack" adhoc with the above change that would just reference the attribute directly in the template that renders the ProxySQL configuration file:

```
mysql> SELECT @@mysql-throttle_connections_per_sec_to_hostgroup;
+---------------------------------------------------+
| @@mysql-throttle_connections_per_sec_to_hostgroup |
+---------------------------------------------------+
| 100                                               |
+---------------------------------------------------+
1 row in set (0.00 sec)
```